vip-helpers/vip-stats.php

```diff
@@ -68,7 +68,7 @@ function wpcom_vip_top_posts_array( $num_days = 30, $limit = 10, $end_date = fal
  * @param int          $post_id   Optional. The post ID to fetch stats for. Defaults to the $post global's value.
  * @param int          $num_days  Optional. How many days to go back to include in the stats. Default is 1. Maximum 90 days.
  * @param string|bool  $end_data  Optional. The last day of the desired time frame. Format is 'Y-m-d' (e.g. 2007-05-01) and default is today's UTC date.
- * @return int|false Number of pageviews or false on error.
+ * @return int Number of pageviews, 0 if empty/error.
```

Comment:
The annotation changes but #L87-89 still `return false;`

Reply:
Ah, nice catch. Do think it would have been better to just return `0` on error, but a bit late now I guess.